tokenizers/src/tokenizer/mod.rs

```diff
@@ -402,6 +402,10 @@ impl Tokenizer {
         let tokenizer = serde_json::from_str(&content)?;
         Ok(tokenizer)
     }
+    pub fn from_bytes<P: AsRef<[u8]>>(bytes: P) -> Result<Self> {
+        let tokenizer = serde_json::from_slice(bytes.as_ref())?;
+        Ok(tokenizer)
```

Comment:
NIT: `serde_json::from_slice(bytes.as_ref())` is IMO slightly better ;)

Reply:
I tried `serde_json::from_slice(bytes.as_ref())`, but rust compiler reports error:

```
mismatched types
expected enum `std::result::Result<tokenizer::Tokenizer, std::boxed::Box<(dyn std::error::Error + std::marker::Send + std::marker::Sync + 'static)>>`
   found enum `std::result::Result<_, serde_json::Error>`
```